origami.json

```
@@ -110,6 +110,17 @@
 				"master"
 			]
 		},
+		{
+			"name": "master-brand-palette",
+			"title": "Brand Palette",
+			"data": "demos/src/palettes/master-palette.json",
+			"template": "demos/src/brand-palette.mustache",
+			"description": "These colors are specifically designed for use in in some digital assets, such as the FT logo. For example, brand-ft-pink could be used to prevent a flash of the wrong color as the logo loads",
```

Removes double "in" and switches to [British English for documentation](https://origami.ft.com/docs/principles/tone-and-language/#use-british-english-for-documentation).
```suggestion
			"description": "These colours are specifically designed for use in some digital assets, such as the FT logo. For example, brand-ft-pink could be used to prevent a flash of the wrong colours as the logo loads",
```

---

The other demo descriptions use `colors` 😱